src/features/fireWeather/components/graphs/TempRHGraph.styles.ts

```diff
@@ -12,7 +12,7 @@ export const modelSummaryRHAreaColor = '#94ffeb'
 export const highResModelTempColor = '#a017c2'
 export const highResModelRHColor = '#3ac417'
 export const highResModelSummaryTempAreaColor = '#ddbae9'
-export const highResModelSummaryRHAreaColor = '#6bbbf5'
+export const highResModelSummaryRHAreaColor = '#b5f0a5'
```

Comment:
Good catch matching the colours. You probably want to check for colourblindness issues again now that it's using the correct colour.

Reply:
Checked and done